content/en/docs/reference/glossary/remote-cluster.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Remote Cluster
+test: n/a
+---
+
+A remote cluster is a cluster that uses a remote profile to connect to a
+control plane residing outside of the cluster. The control plane can reside
+either in another cluster or external to the mesh as is the case with managed
```

Comment:
We can link to the definition of primary cluster here right? A remote cluster can connect to a control plane running in a primary cluster or to a managed control plane.

We also have a glossary entry for managed control plane.